src/Sylius/Bundle/CoreBundle/Routing/RouteProvider.php

```diff
@@ -222,4 +223,26 @@ private function createRouteFromEntity($entity, $value = null)
 
         return new Route($this->routeConfigs[$className]['prefix'].'/'.$value, $defaults);
     }
+
+    /**
+     * @param string $identifier
+     * @param RepositoryInterface $repository
+     * @param string $className
+     *
+     * @return Object|null
+     */
+    private function tryToFindEntity($identifier, RepositoryInterface $repository, $className)
```

Comment:
This is useless as you call magic method instead of existing one which internally will generate same code that was used before.